js/ts_global.d.ts

```diff
@@ -4,8 +4,13 @@
 // when building Deno, but the `typescript/lib/typescript.d.ts` is defined as a
 // module.
 
+// Warning! This is a magical import. We don't want to have multiple copies of
```

Comment:
I don't understand this comment. This is typescript code, what does cargo have to do with it?

Reply:
cargo package must have all dependent files present, in this subtree to work.

Reply:
So the part I don't understand is: how does cargo even know this is a dependent file?